tools/llcstat.py

```diff
@@ -98,22 +117,39 @@
 
 miss_count = {}
 for (k, v) in b.get_table('miss_count').items():
-    miss_count[(k.pid, k.cpu, k.name)] = v.value
-
-print('PID      NAME             CPU     REFERENCE         MISS    HIT%')
+    if args.tid:
+        miss_count[(k.pid, k.tid, k.cpu, k.name)] = v.value
+    else:
+        miss_count[(k.pid, k.cpu, k.name)] = v.value
+
+header_text = 'PID      NAME             CPU     REFERENCE         MISS    HIT%'
+format_text = '{:<8d} {:<16s} {:<4d} {:>12d} {:>12d} {:>6.2f}%'
+if args.tid:
```

Comment:
Can all this printing code be refactored to avoid so many `if args.tid` everywhere?

Something like 
```python
header_text = 'PID   '
if args.tid:
  header_text += 'TID   '
header_text += 'NAME       CPU ....'
```
and similar gradual buildup of the formatted output?

This way when further optional args are added which change the output format it'll be straightforward to incorporate them